src/frontend/src/components/common/Table/views/useTableRowView.tsx

```diff
@@ -0,0 +1,84 @@
+import { faPlus, faTrash } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import InputText from '../../InputText/InputText';
+import { TableViewProps } from './table-view-props.interface';
+
+type UseTableRowViewProps = TableViewProps;
+
+export const UseTableRowView = (props: UseTableRowViewProps) => {
+  return (
+    <>
```

Comment:
If we already have a parent `<div>` in this component, do we really need the `<>` fragment?